src/main/java/br/codenation/hospital/manager/model/BancoSangue.java

```diff
@@ -0,0 +1,9 @@
+package br.codenation.hospital.manager.model;
+
+public class BancoSangue extends ItemEstoque {
```

Comment:
No portuguese in code! It's not a good practice and adds more confusion since all languages have defintions in english